### ISSUE TITLE
Accessibility - ARIA hidden element

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#65]
Homepage: https://qdr.syr.edu/
"ARIA hidden element must not contain focusable elements" (element hidden from screen reader)

![image](https://user-images.githubusercontent.com/89411909/131691015-23e86b1c-b8ef-4248-9dfd-332d00aa665e.png)

**Comment by adam3smith** [comment]
Remove aria-hidden label, add aria-label for search

**Comment by qqmyers** [comment]
The aria-hidden="true" appears to be being added by font awesome (removing the class="fas fa-search" attribute gets rid of the aria-hidden). Because of that, I don't see a way to add an aria-label directly. 

Looking at https://fontawesome.com/v5.15/how-to-use/on-the-web/other-topics/accessibility, they suggest that adding a title is a way of making the icon accessible. 

I verified that adding one creates a <span class="sr-only"> element after the icon. However that alone doesn't resolve the Axe issue so I also added a span containing the icon and put the tabindex="0" attribute on it. That appears to resolve the AXE issue. I also downloaded a screen reader (NVDA) and verified that when you hover over the icon, the sr-only element is read. 

The only side effect is that with a title="Search", there is a pop-up with that text when you hover. 

This change is on dev - let me know there are better ideas/changes.
